travelpal/lib/travelpal_web/controllers/flight_controller.ex

```diff
@@ -1,6 +1,14 @@
 defmodule TravelpalWeb.FlightController do
   use TravelpalWeb, :controller
 
+  # @TODO decide if other functions are needed
+  def flight_url, do: "https://api.skypicker.com/flights"
+
+  def get_flights_to_from(conn, %{"origin" => origin, "dest" => dest, "date_from" => date_from, "return_from" => return_from}) do
+    # gets top 5 flights
+    uri = URI.encode(flight_url() <> "?flyFrom=#{origin}&to=#{dest}&date_from=#{date_from}&date_to=#{date_from}"
+        <> "&return_from=#{return_from}&return_to=#{return_from}&partner=picky&partner_market=us&curr=USD&limit=5")
+    
   alias Travelpal.Flights
```

Comment:
I think these should be moved to the top, under `use TravelpalWeb, :controller`.